packages/desktop/views/dashboard/collectibles/components/Erc721CollectibleDetails.svelte

```diff
@@ -13,7 +13,7 @@
     const address = contractMetadata.address
     const explorerEndpoint = getExplorerEndpoint(networkId)
 
-    function getExplorerEndpoint(networkId: NetworkId): string | undefined {
+    function getExplorerEndpoint(networkId: EvmNetworkId): string | undefined {
```

Comment:
Add the network property as EvmNetworkId on IErc721Nft to improve typing in this file.